journal/src/main/scala/com/github/j5ik2o/akka/persistence/dynamodb/serialization/ByteArrayJournalSerializer.scala

```diff
@@ -75,7 +77,7 @@ class ByteArrayJournalSerializer(
       index: Option[Int]
   )(implicit ec: ExecutionContext): Future[JournalRow] = {
     val pid        = PersistenceId(persistentRepr.persistenceId)
-    val context    = MetricsReporter.newContext(UUID.randomUUID(), pid)
+    val context    = Context.newContext(UUID.randomUUID(), pid)
     val newContext = metricsReporter.fold(context)(_.beforeJournalSerializeJournal(context))
 
     val future = for {
```

Comment:
Change to def.

Reply:
fixed.
https://github.com/j5ik2o/akka-persistence-dynamodb/pull/943/commits/91e2814a4fcf841b337569bf886e735fd5f6d93e#diff-599e2fb130750a3c53ab98d064e4eddec7688c71eb9548e4074946a727aace74R83